### ISSUE TITLE
color flash upon startup & additional empty char on statusbar

### ISSUE THREAD

**Issue by wael444** [martanne/vis#1032]
https://user-images.githubusercontent.com/47404953/183268280-bf93a004-7f02-4033-84bb-29c0b0565060.mp4

**Comment by MaxGyver83** [comment]
Are you using an alias?
Could you try another terminal emulator? Or a another shell?

**Comment by wael444** [comment]
> Are you using an alias?

No

> Could you try another terminal emulator?

Change: there is an entirely useless bar in stock alacritty settings, also still a blank character:
![image](https://user-images.githubusercontent.com/47404953/183295270-2be7fd22-92a8-4023-b393-1e7b2d88810d.png)


> Or another shell?

No change.

**Comment by MaxGyver83** [comment]
OK, it's strange. Here, it doesn't flicker.

Regarding the "empty char":
Do you mean the space next to the `1`? I think, it's just padding. There is also one blank next to the line and column number in vim.
Or do you mean the black area right to the bar? I think this also some kind of padding. When your window has a width of 170.7 characters, then the terminal uses the space needed for 170 characters and leaves the rest (0.7 character width) black. At least [st](https://st.suckless.org/) does it like that.

**Comment by wael444** [comment]
> OK, it's strange. Here, it doesn't flicker.

on stock config it occurs, here is my config:

```lua
require('vis')

vis.events.subscribe(vis.events.WIN_OPEN, function(win)
        vis:command('set theme base16-gruvbox-dark-hard')
end)
```
if it matters, i compiled vis with these flags: `-O3 -pipe -march=x86-64-v3 -mtune=generic -fno-math-errno -fstack-protector-strong --param ssp-buffer-size=4 -D_FORTIFY_SOURCE=2`

**Comment by leorosa** [comment]
Can you check if the flicker also occurs when setting your theme earlier? I.e.
```
vis.events.subscribe(vis.events.INIT, function()
        vis:command('set theme base16-gruvbox-dark-hard')
end)
```

**Comment by wael444** [comment]
still occurs:
```lua
require('vis')

vis.events.subscribe(vis.events.INIT, function()
        vis:command('set theme base16-gruvbox-dark-hard')
end)
```

**Comment by wael444** [comment]
is there a way i can get around this? i could probably pull of ignoring this but this really throws off my eyes each time i launch vis.

**Comment by wael444** [comment]
solved in 0.8
